packit/cli/utils.py

```diff
@@ -30,7 +30,8 @@
 
 from ogr.parsing import parse_git_repo
 from packit.api import PackitAPI
-from packit.config import get_local_package_config, Config
+from packit.config.config import Config
+from packit.config.package_config import get_local_package_config
```

Comment:
You can use something like [this](https://github.com/packit-service/ogr/blob/master/ogr/services/github/__init__.py) in the `packit/config/__init__.py` to not break the imports.

(Otherwise, we need to change that in the packit-service as well.)